tests/e2e/e2e_util_test.go

```diff
@@ -0,0 +1,108 @@
+package e2e
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ory/dockertest/v3/docker"
+)
+
+func (s *IntegrationTestSuite) connectIBCChains() {
+	s.T().Logf("connecting %s and %s chains via IBC", s.chainA.id, s.chainB.id)
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+	defer cancel()
+
+	exec, err := s.dkrPool.Client.CreateExec(docker.CreateExecOptions{
+		Context:      ctx,
+		AttachStdout: true,
+		AttachStderr: true,
+		Container:    s.hermesResource.Container.ID,
+		User:         "root",
+		Cmd: []string{
+			"hermes",
+			"create",
```

Comment:
Note that we plan to change the `create channel` invocation in Hermes. From

`create channel <CHAIN_A> <CHAIN_B> --port-a=transfer --port-b=transfert`

to 

`create channel <CHAIN-A-ID> <CONNECTION-A-ID> --port-a <PORT-ID> --port-b <PORT-ID>`.

In other words, you need to (1) `create connection` and then (2) pass the connection identifier when you invoke `create channel`. This mean you would need to adapt the CI to add an additional CLI command for creating a new connection, as follows:

`create connection s.chainA.id s.chainB.id`.

Assuming the chains are newly-spawned, the connection identifier will be `connection-0` on both chains. Then you would need to invoke create channel as follows:

`create channel s.chainA.id connection-0 --port-a=transfer --port-b=transfer`

Reply:
PS. This change was motivated by relayer operators abusing the command (we got complaints that people create many channels in production with varying client security parameters, ref: https://github.com/informalsystems/ibc-rs/issues/1421).

Would be glad to take your requirements into account and do stuff differently if you have suggestions to change this.

Reply:
This seems like a great change to me! Thanks for letting us know / good design decision!

Reply:
Great to hear it won't affect your setup adversely.

The change will land in Hermes v0.14, slated for end of April.